rust-hyper/src/main.rs

```diff
@@ -2,54 +2,98 @@ extern crate futures;
 extern crate hyper;
 extern crate net2;
 extern crate num_cpus;
+#[macro_use]
+extern crate serde_derive;
+extern crate serde_json;
 extern crate tokio_core;
 
-/*
-This is adapted from the Techempower Hyper benchmark: https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Rust/hyper
```

Comment:
This seems like useful information - why was it removed?

Reply:
I think it got removed during code shuffling. Added again.